src/directives/Directive.ts

```diff
@@ -37,6 +37,7 @@ export abstract class Directive {
 
 	constructor(flag: Flag, colonyFilter?: (colony: Colony) => boolean) {
 		this.memory = flag.memory;
+		this.overlords = {};
```

Comment:
what is the purpose of moving `this.overlords = {};` to this line?

Reply:
https://github.com/bencbartlett/Overmind/blob/master/src/directives/colony/colonize.ts#L45

`spawnMoarOverlords` gets called even if this constructor fails, which causes an exception. This just results in the exception not happening, which doesn't seem to have other bad side-effects.